tools/provision.ps1

```diff
@@ -346,7 +346,7 @@ function Install-ThirdParty {
   # List of our third party packages, hosted in our AWS S3 bucket
   $packages = @(
     "aws-sdk-cpp.1.2.7",
-    "boost-msvc14.1.66.0",
+    "boost-msvc14.1.66.1",
```

Comment:
For changes like these, where we're just updating something on our end and it's not an actual library update, I use a `-r1` or the like for the chocolatey version so we don't have versioning conflicts with subsequent boost iterations.